### PR TITLE
Make map work again

### PR DESCRIPTION
**Pull request by michielderoos**
**Describe the Pull Request**

The map was using only showing 50 transactions. Now all transactions are shown!  
Also adds filter module!

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes, checking [accessibility](https://www.a11yproject.com/checklist/) etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver

### CODE REVIEW

**Comment by michielderoos** [comment]
[Make sure map isn't just loading last 100 txns](https://trello.com/c/qqGQ9Ihj/694-make-sure-map-isnt-just-loading-last-100-txns)

**Comment by michielderoos** [comment]
[Adds filters to Map page](https://trello.com/c/1DlUvA8H/488-adds-filters-to-map-page)

File: app/client/components/pages/mapPage.jsx [hunk 1]
```diff
@@ -1,97 +1,41 @@
...
+    return (
+      <WrapperDiv>
         <PageWrapper>
-          <NoDataMessage />
+          <FilterModule
+            defaultGroupBy={"sender,coordinates"}
+            filterObject={"credit_transfer"}
+            dateRange={null}
+            hideGroupBy={true}
+          />
```
Comment (tristanhcole) [inline app/client/components/pages/mapPage.jsx:29]: Small change to improve styling
Screen Shot 2020-12-15 at 5 48 14 pm

File: app/client/components/heatmap/heatmap.jsx [hunk 2]
```diff
@@ -5,22 +5,25 @@ import { connect } from "react-redux";
...
 import DeckGLOverlay from "./deckglOverlay.jsx";
 
-// Set your mapbox token here
-const MAPBOX_TOKEN =
-  "pk.eyJ1Ijoic2VtcG9uaWNrIiwiYSI6ImNqZnJtNHkybzA0OWMyd25lczcyeXJwMTYifQ.0CJmw4sMU_VuX4wsPlb53Q"; // eslint-disable-line
-
-// Source data CSV
-const DATA_URL =
-  "https://raw.githubusercontent.com/uber-common/deck.gl-data/master/examples/3d-heatmap/heatmap-data.csv"; // eslint-disable-line
-
```
Comment (tristanhcole) [inline app/client/components/heatmap/heatmap.jsx:15]: 👍

**Review by tristanhcole — APPROVED** [review]
I've made a small styling change. Let's MAKE MAP WORK AGAIN! 😂

**Pull request merged**